nrfx_glue.h

```diff
@@ -257,8 +257,15 @@ extern const uint32_t z_bt_ctlr_used_nrf_ppi_groups;
 
 #if defined(CONFIG_PWM_NRF5_SW)
 #define PWM_NRF5_SW_NODE DT_INST(0, nordic_nrf_sw_pwm)
+#define PWM_NRF5_SW_GENERATOR_NODE DT_PHANDLE(PWM_NRF5_SW_NODE, generator)
+#if DT_NODE_HAS_COMPAT(PWM_NRF5_SW_GENERATOR_NODE, nordic_nrf_rtc)
+#define PWM_NRF5_SW_PPI_CHANNELS_PER_PIN	3
+#else /* DT_NODE_HAS_COMPAT(PWM_NRF5_SW_GENERATOR_NODE, nordic_nrf_rtc) */
```

Comment:
This comment does not really help. I think the whole thing will be more readable without it, the one after `#endif` is enough to make everything clear.